vignettes/data-combined.Rmd

```diff
@@ -0,0 +1,300 @@
+---
+title: "Working with `DataCombined` class"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Working with `DataCombined` class}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  dpi = 300,
+  comment = "#>",
+  out.width = "100%",
+  fig.height = 6,
+  fig.width = 8
+)
+```
+
+## Introduction
+
+In Modeling and Simulation (M&S) workflows, we often need to work with multiple 
+observed and/or simulated datasets. Further, some of these datasets can be linked 
+together in a group (e.g., the simulated and the correnspoding observed data)
+for analysis and visualization.
```

Comment:
I assume that every group can include only simulated/observed data with the same dimension?

Reply:
As far as I can recall, this is not required. However, this will produce errors when trying to calculate residuals or create plots... Will add a note about it.

Reply:
shouldn't we introduce a dimension check instead?

Reply:
Issue created
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1132